floyd/cli/run.py

```diff
@@ -147,13 +151,19 @@ def validate_env(env, arch):
     return True
 
 
-def show_new_job_info(expt_client, job_name, expt_info, mode, open_notebook=True):
+def show_new_job_info(expt_client, job_name, expt_info, mode, open_notebook=True, show_data_info=None):
     table_output = [["JOB NAME"], [job_name]]
     floyd_logger.info('\n' + tabulate(table_output, headers="firstrow") + '\n')
 
     job_url = '%s/%s' % (floyd.floyd_web_host, job_name)
     floyd_logger.info("URL to job: %s", job_url)
 
+    if show_data_info is not None:
+        headers = ["DATANAME", "MOUNTING DIRECTORY"]
+        floyd_logger.info('\n' + tabulate(show_data_info, headers=headers))
+        input_data_url = '%s/data' % (job_url)
+        floyd_logger.info("\nURL to Input Data: %s", input_data_url)
```

Comment:
I think this might be redundant given we already print out the job URL. If it's not obvious to find the data tab from the job UI, we should improve the UI instead :)

Reply:
Do you think that it's better to remove this? Any other info that you would like to display?

Reply:
I would prefer to remove this because we are already showing too much stuff on the screen I feel like :D

Reply:
Ok, let's go then with Less is More! 👍